src/System.CommandLine.Subsystems/Subsystems/CliSubsystem.cs

```diff
@@ -28,74 +29,26 @@ protected CliSubsystem(string name, SubsystemKind subsystemKind, IAnnotationProv
     /// </summary>
     public SubsystemKind SubsystemKind { get; }
 
-    private DefaultAnnotationProvider? _defaultProvider;
     private readonly IAnnotationProvider? _annotationProvider;
 
     /// <summary>
-    /// Attempt to retrieve the value for the symbol and annotation ID from the annotation provider.
+    /// Attempt to retrieve the <paramref name="symbol"/>'s value for the annotation <paramref name="id"/>. This will check the
+    /// annotation provider that was passed to the subsystem constructor, and the internal annotation storage.
     /// </summary>
     /// <typeparam name="TValue">The value of the type to retrieve</typeparam>
     /// <param name="symbol">The symbol the value is attached to</param>
-    /// <param name="id">The id for the value to be retrieved. For example, an annotation ID for help is description</param>
+    /// <param name="id">
+    /// The identifier for the annotation value to be retrieved.
+    /// For example, the annotation identifier for the help description is <see cref="HelpAnnotations.Description">.
```

Comment:
Is the type name `HelpAnnotation` used by the CLI author? IOW, do CLI authors ever have to type `HelpAnnotations.Description`? If they do, I think we should consider simplifying the name.

Reply:
It's used `IAnnotationProvider` authors and subsystem authors. What would you simplify it to?